service/src/test/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleServiceTest.java

```diff
@@ -53,8 +64,15 @@ public void shouldMakeCorrectCallForCtsc(Boolean toggleState) {
         givenToggle(toggleState);
 
         assertThat(featureToggleService.isCtscEnabled("test name")).isEqualTo(toggleState);
+        featureToggleService.isCtscReportEnabled();
```

Comment:
Should this be part of this test? We have tests for each of the individual methods with this one being for the `isCtscEnabled` method. Should we be doing this additional verification of objects in a different test as it is testing different functionality.

Reply:
I am confused as to why this has added this to every test.

Reply:
In my opinion we should not have this added to any of the current tests. We should have a new test called something along the lines of `shouldNotCarryAttributesBetweenRequests` that can call the 2 ctsc flags and check that the attributes are not carried over between calls.

Reply:
hmm ok, but how would you ensure every methods works as expected? If we call only 2 ctsc flags, we don't get confirmation allocation judge is free from pollution. Unless we are happy with single test + code review process? But I would like somehow to ensure every test sends only what is expected and only it.

Maybe it's matter of adding dummy invocation in before? Or like all dummy invocations to make it easier and not specific. Iterate using reflection :D

Reply:
Ahh I see what you mean, I was confused as to why we were making this call on all methods but I see your point, will have a think and see if there is a nice way to implement it

Reply:
Also this probably wouldn't have been caught even if we had tested the users being passed as we reset the FeatureToggle instance before each test

Reply:
Made [this](https://github.com/hmcts/fpl-ccd-configuration/compare/FPLA-1912-CTSC-Feature-Toggle-Fix...dev/fpla-1912-test-example) as an example of what could be done